src/Parser.php

```diff
@@ -1,13 +1,9 @@
 <?php
-
```

Comment:
can you revert the line changes, it makes the diff incorrect

Reply:
@ab02111996 Agree with this. Can't really merge as-is.